pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "aws_advanced_python_wrapper"
-version = "1.3.0"
+version = "1.4.0"
```

Comment:
Can you also update the version in `driver_info.py` please?

Reply:
Done!